src/components/existing-request/existing-request-display.vue

```diff
@@ -734,6 +775,117 @@ export default class ExistingRequestDisplay extends Mixins(
       this.$root.$emit('showSpinner', false)
     }
   }
+
+  private get isThereMoreThanOnePaymentMethod () {
+    const paymentMethods = this.payments.map(payment => payment.sbcPayment.paymentMethod)
+    return paymentMethods.some(method => method !== paymentMethods[0])
+  }
+
+  private get isNoFeePayment () {
+    if (this.payments.length > 1) {
+      return this.payments.reduce((paymentA, paymentB) => paymentA.sbcPayment.paid + paymentB.sbcPayment.paid) === 0
+    } else if (this.payments.length === 1) {
+      return this.payments[0].sbcPayment.paid === 0
+    }
+    return true
+  }
+
+  private get isNoRefund () {
+    if (this.payments.length > 1) {
+      return this.payments.reduce(
+        (paymentA, paymentB) => paymentA.sbcPayment.refund + paymentB.sbcPayment.refund
+      ) === 0
+    } else if (this.payments.sbcPayment.length === 1) {
+      return this.payments[0].refund === 0
+    }
+    return true
+  }
+
+  private get isRefundRequested () {
+    if (this.nr.state === NrState.REFUND_REQUESTED) {
+      this.buildRefundParams()
+      return true
+    }
+    return false
+  }
+
+  private buildRefundParams () {
+    if (this.nr.state === NrState.REFUND_REQUESTED) {
+      if (!this.isThereMoreThanOnePaymentMethod) {
+        const paymentMethod = this.payments[0]?.sbcPayment?.paymentMethod
+        if (paymentMethod === PaymentMethod.PAD) {
+          // Premium Account
+          if (!this.isNoRefund) {
+            this.refundParams.refundLabel = 'Refund Request Processed'
+            this.refundParams.refundMessage =
+              'Your Name Request has been cancelled and a refund request is being processed.<br/><br/>' +
+              'A credit will be applied to your BC Registries account.<br/>There may be a one day delay before the ' +
+              'credit will show on your transactions / statetements.'
+            this.refundParams.showStaffContact = false
+          } else {
+            // May happen when a PAD is not processed yet.
+            // It usually takes a day to be processed.
+            this.refundParams.refundLabel = 'Refund Not Processed'
+            this.refundParams.refundMessage =
+              'Your Name Request has been cancelled.<br/><br/>' +
```

Comment:
Why the double <br>?

Reply:
![image](https://user-images.githubusercontent.com/78114138/129979972-8e175877-5d01-4e50-b072-c3428c10f41a.png)
 There's an empty line.

Reply:
Easy!